### ISSUE TITLE
Exception in  plugin Catdea(1.1.2)

### ISSUE THREAD

**Issue by Z-J-Fei** [Cybr0sis/Catdea#11]
```
   com.intellij.diagnostic.PluginException: Incorrect language specified in <fileType> for Catdea, should be CatdeaLanguage, actual null [Plugin: com.cybrosis.catdea]
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.instantiateFileTypeBean(FileTypeManagerImpl.java:497)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.instantiatePendingFileTypeByName(FileTypeManagerImpl.java:659)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.getFileTypeByName(FileTypeManagerImpl.java:1616)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.readGlobalMappings(FileTypeManagerImpl.java:1500)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.loadState(FileTypeManagerImpl.java:1392)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.loadState(FileTypeManagerImpl.java:85)
	at com.intellij.configurationStore.ComponentStoreImpl.doInitComponent(ComponentStoreImpl.kt:412)
	at com.intellij.configurationStore.ComponentStoreImpl.initComponent(ComponentStoreImpl.kt:363)
	at com.intellij.configurationStore.ComponentStoreImpl.initPersistenceStateComponent(ComponentStoreImpl.kt:130)
	at com.intellij.configurationStore.ComponentStoreImpl.initComponent(ComponentStoreImpl.kt:96)
	at com.intellij.configurationStore.ComponentStoreWithExtraComponents.initComponent(ComponentStoreWithExtraComponents.kt:49)
	at com.intellij.serviceContainer.PlatformComponentManagerImpl.initializeComponent(PlatformComponentManagerImpl.kt:258)
	at com.intellij.serviceContainer.ServiceComponentAdapter.createAndInitialize(ServiceComponentAdapter.kt:56)
	at com.intellij.serviceContainer.ServiceComponentAdapter.doCreateInstance(ServiceComponentAdapter.kt:39)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstanceUncached(BaseComponentAdapter.kt:110)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstance(BaseComponentAdapter.kt:72)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstance$default(BaseComponentAdapter.kt:65)
	at com.intellij.serviceContainer.PlatformComponentManagerImpl$preloadServices$future$1.run(PlatformComponentManagerImpl.kt:622)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1736)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:222)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:30)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.execute(BoundedTaskExecutor.java:201)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:193)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
	```

**Comment by BlackElkX** [comment]
This was reporded already 7 days ago.
See also https://github.com/Cybr0sis/Catdea/issues/10

**Comment by Cybr0sis** [comment]
Duplicate of issue #9 
Fixed in [Catdea release v1.1.3](https://github.com/Cybr0sis/Catdea/releases/tag/v1.1.3)
